web/src/App.js

```diff
@@ -1,26 +1,267 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import Card from '@material-ui/core/Card';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+import Web3 from 'web3';
+
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+const { abi, networks } = require('./contracts/Splitter.json');
+
+// Support ropsten testnet
+const contractAddress = networks['3'].address;
+
+class App extends Component {
+  state = {
+    owner: '',
+    balances: [],
+    hasError: false,
+    message: '',
+    contractLoaded: false,
+    accountAddress: '',
+    etherToSplit: '1',
+    beneficiary1Address: '',
+    beneficiary2Address: '',
+  };
```

Comment:
This list does not include `contract` and `accounts` and others as seen on line 56. Is it normal?